components/gif_picker/components/InfiniteScroll/index.ts

```diff
@@ -105,7 +106,7 @@ export default class InfiniteScroll extends PureComponent {
 
             // Call loadMore after detachScrollListener to allow for non-async loadMore functions
             if (typeof this.props.loadMore === 'function') {
```

Comment:
loadMore is marked as required and is expecting a function, so the check for the type of this.props.loadMore is not needed here.

Reply:
Then shall I go ahead and remove that check? @m1lt0n